### PR TITLE
added FullHD, 4K streams support for getUserMedia via 'fhdres' and '4kres' presets

### PR DESCRIPTION
**Pull request by soulfly**
I want to use FullHD cameras with Janus, so this pull requests address it.
Also 4K support is added

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -53,3 +53,6 @@ Makefile.in
 
 .deps
 .dirstamp
+
+# OS X
+.DS_Store
```
Comment (lminiero) [inline .gitignore]: What's this?

Comment (soulfly) [reply .gitignore]: I'm developing on macOS. This is kind of standard file generated almost in each macOS folder, something like desktop.ini in Microsoft Windows. https://en.wikipedia.org/wiki/.DS_Store
It's normal to add it to gitignore

**Comment by lminiero** [comment]
Looks good! Can you filling in the form in our [CLA](https://cla.conf.meetecho.com/janus/) so that I can merge? (although I thought you had already done that) Thanks!

**Comment by lminiero** [comment]
Thanks, merging then! :+1:

**Pull request merged**